packages/docs/docs/getting-started/user-interface.mdx

```diff
@@ -56,7 +55,13 @@ After [launching](/docs/getting-started/launching) the the Radon IDE panel the u
     <img width="400" src="/img/docs/ide_zoom_controls.png" />
 11. **Device preview** - the preview of the running iOS simulator or Android emulator.
 12. [**Element inspector**](/docs/features/element-inspector) - enabling element inspector and clicking on the preview opens the line of code where the highlighted component is defined. Right-clicking on the device preview allows you to select components directly from the React tree.
-13. **App root selector** - lets you switch between applications in a monorepo setup.
+13. **Launch configuration selector** - lets you switch between different [launch configurations](/docs/guides/configuration) for your workspace.
+
+    <img width="250" src="/img/docs/ide_approot_select.png" className="shadow-image" />
+    In a monorepo workspace, the drop-down displays all automatically detected React Native applications.
```

Comment:
I wouldn't start from mentioning monorepo here as users with uni-repo structure would stop reading immediately :)

Perhaps would start with just describing this option here:
> This menu lets you configure and launch different flavors of your applications (i.e. staging configuration) or switch between different app root directories in monorepo setups.